src/libostree/ostree-repo.c

```diff
@@ -5099,3 +5693,47 @@ ostree_repo_set_collection_id (OstreeRepo   *self,
 
   return TRUE;
 }
+
+/**
+ * ostree_repo_get_lock_timeout:
+ * @self: an #OstreeRepo
+ *
+ * Get the current lock timeout for the repository. A timeout of -1 indicates
+ * that the application will block until the lock is acquired. Otherwise, it
+ * represents the number of seconds the application will sleep attempting to
+ * acquire the repository lock.
+ *
+ * Returns: lock timeout for the repository
+ * Since: 2017.13
+ */
+gint
+ostree_repo_get_lock_timeout (OstreeRepo *self)
+{
+  return self->lock_timeout;
+}
+
+/**
+ * ostree_repo_set_lock_timeout:
+ * @self: an #OstreeRepo
+ * @timeout: new lock timeout in seconds
+ *
+ * Set the lock timeout for the repository. A timeout of -1 indicates that the
+ * application will block until the lock is acquired. Otherwise, it represents
+ * the number of seconds the application will sleep attempting to acquire the
+ * repository lock.
+ *
+ * Note that if the repository configuration is reloaded, then a timeout
+ * configured in the core.lock-timeout option will replace this.
+ *
+ * Returns: %TRUE on success, %FALSE otherwise
+ * Since: 2017.8
```

Comment:
Since: 2017.13

(probably wrong in a few other places)

Reply:
Bah. I actually think I was pretty good about setting it to 2017.13, but I'll skim through for that. Thanks for catching that.